app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
+  &.margin {
+    margin: 0 !important;
+  }
+
+  &.bottom.margin {
+    margin-bottom: 0 !important;
+  }
+
+  &.top.margin {
+    margin-top: 0 !important;
+  }
 }
 
 .less {
   &.padding {
     padding: 0 !important;
   }
-  
-  &.padding-right {
+
+  &.right.padding {
```

Comment:
@niranjan Do I need to write similar code for padding as well or the one which is there right now is correct? please clarify it so that I can make all the changes.